go.mod

```diff
@@ -553,3 +553,5 @@ replace (
 	github.com/heroku/docker-registry-client => github.com/stackrox/docker-registry-client v0.2.0
 	github.com/mholt/archiver/v3 => github.com/anchore/archiver/v3 v3.5.2
 )
+
+replace github.com/quay/claircore => github.com/Maddosaurus/claircore v0.0.0-20240813125501-08eee60eff09
```

Comment:
A leftover?

Reply:
No, unfortunately I discovered we need one more PR for the ClairCore folks - the error handling for the Whiteout and RPM scanner aren't merged.
That should be a quick PR, hopefully though.

Reply:
Update: Both scanners only scan `/host/usr/share/*` - we can get away with only mounting and scanning that subfolder without the scanners erroring out.
This conveniently cuts the scan duration by 90%.
The only downside is that we will have the host filesystem mounted twice in compliance, once in `/host` for legacy compliance checks, and once somewhere for the v4 index, e.g. `/hostindex` (TBD).